op-chain-ops/genesis/config.go

```diff
@@ -217,6 +217,9 @@ type DeployConfig struct {
 	// RequiredProtocolVersion indicates the protocol version that
 	// nodes are recommended to adopt, to stay in sync with the network.
 	RecommendedProtocolVersion params.ProtocolVersion `json:"recommendedProtocolVersion"`
+
+	// When Cancun activates. Relative to L1 genesis.
+	L1CancunTimeOffset *uint64 `json:"l1CancunTimeOffset,omitempty"`
```

Comment:
For my understanding, why don't we need to add a `L1ShanghaiTimeOffset`?

Reply:
We always enable Shanghai from the L1 genesis block so it doesn't have its own config.